lib/components/logIn.dart

```diff
@@ -253,8 +290,55 @@ class _LogInScreenState extends State<LogInScreen> {
                   ],
                 ),
                 SizedBox(
-                  height: 20.0,
+                  height: height * 0.02,
+                ),
+                Container(
+                  width: MediaQuery.of(context).size.width - 50,
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      Expanded(
+                          child: Divider(
+                        color: Colors.grey,
+                      )),
+                      Text(
+                        "Or Connect",
+                        style: TextStyle(color: Colors.grey),
+                      ),
+                      Expanded(
+                          child: Divider(
+                        color: Colors.grey,
+                      )),
+                    ],
+                  ),
+                ),
+                SizedBox(height: 15),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    GoogleAuthButton(
+                        borderRadius: 29,
+                        iconSize: 30,
+                        style: AuthButtonStyle.icon,
+                        onPressed: () async {
+                          await _auth.signInWithGoogle().then(
```

Comment:
you should either use `await` or `then` not both, 

what you should be doing is 

```
final result =  await _auth.signInWithGoogle();

 if (result != null) {
                                _image = await urlToFile(_auth.imageUrl);
                                await uploadProfilePic(context);
....

```